test/test_input/test_shallow_water_config.yml

```diff
@@ -11,6 +11,12 @@ model:
   b0: 4.0
   h0: 5.0
 
+runtime:
```

Comment:
None of the tests use this section of the namelist.  So this won't do anything.  The main program and visualization will fail with this YAML file because the timestep is not valid.  This was not intended for testing the full integration, but rather just the configuration classes.